Makefile

```diff
@@ -1,14 +1,11 @@
 .PHONY: quality style
 
 # Check that source code meets quality standards
-
 quality:
-	black --check . src
-	isort --check-only . src
-	flake8 . src
+	black --check --diff --preview .
+	ruff .
 
 # Format source code automatically
-
 style:
-	black . src
-	isort . src
\ No newline at end of file
+	black --preview .
```

Comment:
Let's revert this line back, if it doesn't break anything. See transformers [equivalent](https://github.com/huggingface/transformers/blob/4deaa534f54b4a1ce4fb6496af1f603229b157c6/Makefile#L68-L70).

Also will ruff work recursively here or are we missing `src` in addition to `.`?

Reply:
Indeed the `--preview` flag is mainly for string formatting, but sure, that's not stable and should be removed. And yes, indeed `black` does the exploration recursively as `ruff`, so setting it to `.` means all the nested directories will be formatted/linted if Python files.

Reply:
awesome I didn't know that. thanks :)